### ISSUE TITLE
I can't insert new images as layers on only some files

### ISSUE THREAD

**Issue by mysticalwillow** [photopea/photopea#4899]
When I've been working on a graphic for some time, I will be unable to insert any images as layers, whether it's through copy and pasting or getting it from my PC files. I can't save my work either. It says that there has been an error and to contact the support email, but when I try to insert the same exact image into a brand new file, everything works fine.

**Comment by photopea** [comment]
Could you show us a file which you can not open in Photopea? Or describe any steps how can we make such problem happen?

**Comment by mysticalwillow** [comment]
It is not a specific file that doesn't open, but it's pretty much every single one. What's weird is that the same images work when I'm trying to paste them into a brand new file, but not on files I've been working on for a while. When this happens, exporting the file in any type doesn't work either. I'm not sure how this problem can occur. I've been facing this problem since yesterday. Everything worked fine the other day.

**Comment by Bees1234** [comment]
I have the same problem, and still no fix. I am very frustrated at this problem. It is the first time it has happened.

Just a question, which browser did you install photopea from if you have, or which browser do you use?

**Comment by photopea** [comment]
Could you try this? https://www.photopea.com/tuts/photopea-doesnt-work-what-to-do/

**Comment by Bees1234** [comment]
My google chrome wasn't updated fully and when I tried to click new it worked!

Thanks for the help :)

**Comment by mysticalwillow** [comment]
I figured out what was causing the issue. 
I tried all of the instructions from the tutorial that was linked, but they didn't really help. I tried to play around in the program, testing what could've made the problem appear and I found out that it was the extension of the file or, rather, the lack thereof. When I download images off Discord, which is a communicator I often use, the file automatically doesn't have an extension (in properties of the file it quite literally says 'File: file' when in PNG, it says 'File: file.png'.), so it is necessary to write an extension at the end manually. 
If you do not do so and choose to try and insert the file as a layer into Photopea, there are a few things that happen. 
- The layer is named -1, regardless of the actual name of the file.
- Exporting the file is either no longer possible.
- Sometimes duplicating layers isn't possible as well.
If the layer is removed, everything goes back to normal, except you still can't export the file. 
This, I must say, is quite peculiar, and while it's easy to work around to elude the issue entirely, once you insert the file without an extension, whatever you do, you cannot export the file you're working on.
